app/src/main/AndroidManifest.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.example.myrecipepres">
-
+    <uses-permission android:name="android.permission.INTERNET" />
```

Comment:
Adding to this branch in case we want to show off the RecipeDiscoveryScreen with mock data, since the RecipeCard uses AsyncImage which needs to get the images from Mr. Internet!